lib/services/environment.py

```diff
@@ -0,0 +1,63 @@
+from typing import Self
+
+import dill
+
+from rocketpy.environment.environment import Environment as RocketPyEnvironment
+from rocketpy.utilities import get_instance_attributes
+from lib.models.environment import Env
+from lib.views.environment import EnvSummary
+
+
+class EnvironmentService:
+    _environment: RocketPyEnvironment
+
+    def __init__(self, environment: RocketPyEnvironment = None):
+        self._environment = environment
+
+    @classmethod
+    def from_env_model(cls, env: Env) -> Self:
+        """
+        Get the rocketpy env object.
+
+        Returns:
+            RocketPyEnvironment
+        """
+        rocketpy_env = RocketPyEnvironment(
+            latitude=env.latitude,
+            longitude=env.longitude,
+            elevation=env.elevation,
+            date=env.date,
+        )
+        rocketpy_env.set_atmospheric_model(
+            type=env.atmospheric_model_type, file=env.atmospheric_model_file
+        )
+        return cls(environment=rocketpy_env)
```

Comment:
Just take in mind that the `set_atmospheric_model` may not work if you use Forecast models for dates in the past.

In other words, the same code may generate different results depending on the time you execute it, this is due to the `env.date` attribute.

For example, you can only use Forecast/GFS model for dates in the future.

Reply:
Noted!

I think this sums up to a large list of scenarios where simulation would fail due to invalid input, even if in this particular case the input was valid at some point in time.

I've discussed with @phmbressan around such issues and we agreed that it would be great to safe-guard some of the inputs to reduce errors like that. There is a github issue to handle that in the future https://github.com/RocketPy-Team/Infinity-API/issues/26